README.adoc

```diff
@@ -5,6 +5,7 @@
 :forked_source_www: https://vsoch.github.io
 :theme_demo_www: https://asciidocsy.netlify.app
 :theme_docs_www: https://asciidocsy.netlify.app/docs
+:theme_docs_bootstrapping_www: https://asciidocsy.netlify.app/docs/theme/setup/bootstrap
```

Comment:
Thanks for catching this! I think this one would be better named with a hyphen, since it's a specification of the main target, and the value can be based on the previous docs attribute.
```suggestion
:theme_docs-bootstrapping_www: {theme_docs_www}/theme/setup/bootstrap
```